server/app/mutations/grid_services/common.rb

```diff
@@ -84,6 +84,26 @@ def build_grid_service_secrets(existing_secrets)
       service_secrets
     end
 
+    def build_service_volumes()
```

Comment:
Still too complex? Refactored heavily when implemented support for the external volume "mapping"